tests/unit/classes/column-test.js

```diff
@@ -29,6 +29,23 @@ test('create column - column instance', function(assert) {
   assert.equal(col2.label, 'Name');
 });
 
+test('reopen column', function(assert) {
+  Column.reopenClass({
+    someStaticMethod() {
+      return true;
+    }
+  });
+  assert.ok(Column.someStaticMethod());
+
+  Column.reopen({
+    someMethod() {
+      return true;
+    }
+  });
+  let col = new Column();
+  assert.ok(col.someMethod());
+});
+
```

Comment:
Not really sure, if these tests make a lot of sense, since we don't run CI tests for IE anyways.

Also, they mutate the Classes. This means that all following test cases have access to `someStaticMethod` and `someMethod`. This might not be a problem now (and I doubt it ever will), but it could and feels kind of dirty.